src/styles/blocks/body.scss

```diff
@@ -0,0 +1,4 @@
+.body {
+  color: $color-dark;
```

Comment:
Ensure that the variable `$color-dark` is defined in your SCSS files. If it's not defined, it will cause a compilation error.